### DISCUSSION TITLE
[Support] 关于 bull-board 的两个问题

### DISCUSSION THREAD

**Discussion by phoooob** [midwayjs/midway#3629] [Q&A]
### Describe the problem（描述问题）

1. 队列列表中的队列时有时无，难道不是创建好的队列应该一直显示吗？刚点到某个队列查看，一下就闪没了。

![ob1C5Y9peV](https://github.com/midwayjs/midway/assets/13043634/2f769af6-3b73-46fb-91c4-d5148e6edcce)

2. 这个组件的路由，全局的middleware和gurad 对其都无效，怎么能对其生效呢？

### Midway Versions（Midway 版本）

"@midwayjs/bull": "^3.14.9",
 "@midwayjs/bull-board": "^3.14.10",

**Comment by czy88840616** [comment]
1、bull dashboard 展示有他自己的逻辑，这个我回答不了，猜测和清理队列时机或者任务执行完成有关系。
2、这个组件里的并不是路由，而是类似 staticFile 的静态文件输出

**Comment by czy88840616** [comment]
之前的改动有点问题，会影响到别的地方，我先回滚了

- 之前的逻辑中，唯一的清理逻辑是在开启 clearRepeatJobWhenStart 的时候，清理历史的重复执行的任务，看看是不是那个影响的
- egg 的生命周期是强行塞进去的。是在 egg server ready 之后才执行 midway 的 framework 初始化，所以是有一些滞后的


我暂时没有复现的 case，有需要的话你可以提交一个。
